Libraries/Image/RCTGIFImage.m

```diff
@@ -86,7 +86,7 @@
   }
 
   if (![URL isFileURL]) {
-    RCTLogError(@"Loading remote image URLs synchronously is a really bad idea.");
+    RCTLogError(@"Loading a URL synchronously on the main thread will cause poor performance, try loading the URL asynchronously. Go to https://facebook.github.io/react-native/docs/image.html#network-resources");
```

Comment:
I think this blocks the image loader thread but not the main thread so it will block the loading of other images.